src/Farmer/Builders/Builders.Storage.fs

```diff
@@ -400,6 +412,30 @@ type StorageAccountBuilder() =
                   DefaultAction = RuleAction.Deny } |> Some
         }
 
+    /// Disable blob public access
+    [<CustomOperation "disable_blob_public_access">]
+    member _.DisableBlobPublicAccess(state:StorageAccountConfig) =
```

Comment:
I wonder if this could be simplified with an optional argument?
```
?flag = FeatureFlag.Enabled
```

Reply:
I don't know why I didn't do that in the first place. Done :+1: